app/config/parameters.yml.dist

```diff
@@ -19,3 +19,5 @@ parameters:
     secret:            SuperSecretTokenPleaseChangeIt
     swp_updater.version.class: "SWP\WebRendererBundle\Version\Version"
     active_theme: "theme_1"
+    swp_multi_tenancy.tenant.class: SWP\MultiTenancyBundle\Model\TenantInterface
```

Comment:
but we will change it per instance? If not then it can go to parameters but inside config.yml section `parameters`